### PR TITLE
[MIBM-531][JR] copy goodsDestination from original declaration to ame…

### PR DESCRIPTION
**Pull request by jordanrowe**
…nd journey

### CODE REVIEW

File: test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersNewHandlerSpec.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ import play.api.test.Helpers._
 import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
 import uk.gov.hmrc.merchandiseinbaggage.config.MibConfiguration
 import uk.gov.hmrc.merchandiseinbaggage.connectors.{MibConnector, PaymentConnector}
-import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType.{Export, Import}
+import uk.gov.hmrc.merchandiseinbaggage.model.api.DeclarationType.Export
```
Comment (andy1138) [inline test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersNewHandlerSpec.scala:25]: This looks like only an import change, is it needed?

File: test/uk/gov/hmrc/merchandiseinbaggage/smoketests/AdditionalDeclarationImportSpec.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.merchandiseinbaggage.smoketests
 
-import uk.gov.hmrc.merchandiseinbaggage.model.api.{DeclarationId, DeclarationType, SessionId}
+import uk.gov.hmrc.merchandiseinbaggage.model.api.{DeclarationType, SessionId}
```
Comment (andy1138) [inline test/uk/gov/hmrc/merchandiseinbaggage/smoketests/AdditionalDeclarationImportSpec.scala:19]: As above, only an import change, is it needed?

Comment (jordanrowe) [reply test/uk/gov/hmrc/merchandiseinbaggage/smoketests/AdditionalDeclarationImportSpec.scala:19]: compilation warns

File: test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersNewHandlerSpec.scala (hunk 1 above)
Comment (jordanrowe) [reply test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersNewHandlerSpec.scala:25]: compilation warns

Comment (andy1138) [reply test/uk/gov/hmrc/merchandiseinbaggage/controllers/CheckYourAnswersNewHandlerSpec.scala:25]: The compiler knows everything :-)

**Review by andy1138 — APPROVED** [review]

**Pull request merged**